frontend/src/pages/GraphicalSelector/styles.ts

```diff
@@ -32,9 +32,16 @@ const SearchBarWrapper = styled.div`
   right: 20px;
 `;
 
+const ToolsWrapper = styled.div`
+  display: flex;
+  align-items: flex-end;
```

Comment:
do
```
align-items: center;
gap: 10px;
```

Reply:
new look: <img width="485" alt="image" src="https://user-images.githubusercontent.com/97064025/189580362-bf69c9f4-c22f-450a-b487-fcbed947ca63.png">

Reply:
Oh lol I meant replace the `align-items: flex-end` line with the above 😅. So keep in the display, margin and other css styles haha

Reply:
oh lol ok sweet cause i was gonna be like looks a bit dodge rn
xD

Reply:
anyway should be fixed with hte new stuff: 
<img width="493" alt="image" src="https://user-images.githubusercontent.com/97064025/189583392-f3ae1b6e-0a1a-4c8f-b2bf-23ece51950d0.png">